notebooks/week5/03.evaluate_model.py

```
@@ -0,0 +1,170 @@
+2. Prepares test data with feature engineering
+3. Generates predictions from both models
+4. Calculates error metrics
+5. Makes registration decision based on MAE comparison
+6. Updates pipeline task values with results
+"""
+
+from databricks import feature_engineering
+from databricks.sdk import WorkspaceClient
+from pyspark.sql import SparkSession
+from pyspark.sql import functions as F
+from pyspark.sql.functions import col
```

Ruff

`pyspark.sql.functions.col` imported but unused

Remove unused import: `pyspark.sql.functions.col`

(F401)